.travis.yml

```diff
@@ -12,9 +12,9 @@ services:
   - mysql
 
 install:
-  - go get -u github.com/Masterminds/glide
+  - go get -u github.com/golang/dep/cmd/dep
```

Comment:
Can you use the latest released version? So we use the same version during development (`brew install dep`) and in CI.